README.md

```diff
@@ -20,34 +20,42 @@ cd deploy-addons
 # done
 ```
 
+## Kubernetes Config File
+By default, the Bootkube generated Kubernetes configuration will not work with Armada, because the config is not technically valid (Tiller will return an error message). This is because the certificate and key data is `REDACTED`, and because there is no default context configured. In order to create a valid `/home/${USER}/.kube/config` file (for use with [Armada](https://github.com/att-comdev/armada)), you will need to perform the following manual actions:
```

Comment:
not work with Armada, because = i guess i don't need the extra comma here. overuse.

Reply:
you will need to perform the following manual actions = only temporary. could be more clear that this is not a perm action for users.